CommonTools/TrackerMap/src/TrackerMap.cc

```diff
@@ -1762,9 +1762,9 @@ void TrackerMap::save_as_HVtrackermap(bool print_total,float minval, float maxva
 	col =gROOT->GetColor(ncolor+100);
 	if(col) 
 	  col->SetRGB((Double_t)(red/255.),(Double_t)(green/255.),(Double_t)(blue/255.)); 
-	else 
+	else {
 	  c = new TColor(ncolor+100,(Double_t)(red/255.),(Double_t)(green/255.),(Double_t)(blue/255.));
-	  vc.push_back(c);
+	  vc.push_back(c);}
```

Comment:
Please put the closing brace on a separate line. It's quite hard to see otherwise. Same for line 1486.